Sources/OktaIdx/IDXClient.swift

```diff
@@ -43,26 +51,31 @@ public final class IDXClient: NSObject {
     /// Starts a new authentication session using the given configuration values. If the client is able to successfully interact with Okta Identity Engine, a new client instance is returned to the caller.
     /// - Parameters:
     ///   - configuration: Configuration describing the app settings to contact.
-    ///   - state: Optional state string to use within the OAuth2 transaction.
+    ///   - options: Options to include within the OAuth2 transaction.
     ///   - completion: Completion block to be invoked when a client is created, or when an error is received.
     public static func start(with configuration: Configuration,
-                             state: String? = nil,
+                             options: [Option:String]? = nil,
```

Comment:
Is this a breaking API change? (I think it is, which is why I'm asking)

Reply:
This is technically a breaking change, though the 2.0.0 release was just cut recently which introduced even more significant breaking changes.  Since this argument is optional, and very few people will actually use it, I'm okay with just doing a minor version change for this update.